consultation_analyser/consultations/download_consultation.py

```diff
@@ -61,8 +64,12 @@ def consultation_to_json(consultation):
             answer_attrs = select_keys_from_model(
                 answer, ["question", "multiple_choice", "free_text"]
             )
-
-            answer_attrs["theme_id"] = str(answer.theme.id) if answer.theme else None
+            # TODO - for now, the assumption is at most one theme per answer
+            latest_theme = None
+            if latest_processing_run:
+                themes_for_answer = latest_processing_run.get_themes_for_answer(answer_id=answer.id)
+                latest_theme = themes_for_answer.last()
+            answer_attrs["theme_id"] = str(latest_theme.id) if latest_theme else None
```

Comment:
Could indent this and lose `latest_theme = None` above? It will never be assigned if it's not set here

Reply:
Potentially, the `latest_theme` could be none, for example, if the response is empty.

Reply:
Ignore last comment, removing the `latest_theme=None` above and indenting works!

Reply:
Now updated.

Reply:
Has been updated.